src/svg/ds6/program-badges.svg

```diff
@@ -0,0 +1,50 @@
+    </symbol>
+    <symbol id="program-badge-money-back-guarantee-uk" viewBox="0 0 48 48">
+        <defs/>
+        <path fill="#3665F3" fill-rule="evenodd" d="M26.1482.4785l15.398 7.21c1.506.706 2.466 2.238 2.454 3.92l-.1201 15.35c.044 4.992-.8799 8.282-3.5159 11.746-3.446 4.6-8.986 7.88-16.364 9.294-7.378-1.414-12.918-4.694-16.36605-9.294-2.634-3.464-3.558-6.754-3.514-11.746l-.12-15.35c-.014-1.682.946-3.214 2.452-3.92L21.8522.4785c1.362-.638 2.932-.638 4.296 0zm-1.2556 11.5216c-4.2088 0-7.668 3.3286-7.668 7.492 0 .0703.0022.1382.004.1913v3.4273h-1.2257c-.5523 0-1 .4477-1 1s.4477 1 1 1h1.2257v6.8888H14.892c-.5523 0-1 .4477-1 1s.4477 1 1 1h3.3013a1.03453 1.03453 0 00.0706 0H31.558c.5523 0 1-.4477 1-1s-.4477-1-1-1H19.2286v-6.8888h7.8863c.5523 0 1-.4477 1-1s-.4477-1-1-1h-7.8863v-3.4446c0-.0115-.0002-.023-.0006-.0344-.0021-.0609-.0034-.1017-.0034-.1396 0-3.0085 2.5128-5.492 5.668-5.492 3.1548 0 5.666 2.4831 5.666 5.492 0 .5523.4477 1 1 1s1-.4477 1-1c0-4.163-3.4568-7.492-7.666-7.492z" clip-rule="evenodd"/>
+    </symbol>
+    <symbol id="program-badge-money-back-guarantee-us" viewBox="0 0 48 50">
+        <defs/>
+        <path fill="#3665F3" fill-rule="evenodd" stroke="#3665F3" d="M25.9055 1.37253c-1.3052-.52982-2.7713-.4945-4.0532.10597l-15.4 7.21-.1933.09672C4.86466 9.528 3.9869 10.9996 4.00015 12.6085l.12015 15.35-.00127.4631c.01669 4.7347.96358 7.9271 3.51527 11.2829 3.448 4.6 8.988 7.88 16.366 9.294 7.378-1.414 12.918-4.694 16.364-9.294l.2421-.3243c2.4507-3.347 3.3165-6.5857 3.2739-11.4217l.1187-15.4045-.0025-.164c-.0687-1.5952-1.0097-3.0262-2.4502-3.7015l-15.398-7.21-.2428-.10597zm.8703 26.35267c-1.4193-.8138-2.9904-1.3224-4.5475-1.8154-.2822-.0896-.5669-.1741-.8517-.2587-1.2394-.368-2.4812-.7367-3.5301-1.5273C16.6388 23.1535 16 21.6746 16 20.1566c0-2.6094 1.6543-4.6384 4.0245-5.5909.7406-.2993 1.5371-.495 2.3815-.6123l.004-1.4633c.0326-1.9862 3.142-1.9875 3.184 0v1.4496c1.6449.2484 2.8287.9194 4.0285 1.8975.8623.7023 1.4872 1.5004 1.8904 2.4512.4172.982-.3513 2.0677-1.4373 2.097-.6667.0176-1.2696-.3873-1.5052-1.0016-.7186-1.8623-2.0182-2.8893-4.5514-2.8893-2.1014 0-4.807.8345-4.807 3.3491 0 2.2105 1.9577 2.7547 3.7275 3.2467.2568.0714.5097.1417.7521.2158.2737.0837.5493.1648.8252.246.7948.2339 1.5927.4687 2.3588.7693 1.1499.4519 2.3556.9722 3.2878 1.7899 2.0154 1.7704 2.52 4.9133.7387 7.0405-1.3537 1.6092-3.343 2.1221-5.3081 2.5274v1.8056c0 2.0209-3.0982 2.0197-3.0982 0v-1.8056s-1.6566-.3942-2.2298-.6416c-1.4765-.6365-2.7754-1.7232-3.5394-3.1496-.5489-1.0231.2496-2.2574 1.4294-2.2574.6308 0 1.1937.3736 1.4433.9429.8023 1.8243 2.8139 2.5215 4.6892 2.5215 1.8222 0 4.4334-.7861 4.4656-3.0086.018-1.0975-1.1059-1.8585-1.9783-2.3612z" clip-rule="evenodd"/>
+    </symbol>
+    <symbol id="program-badge-money-back-guarantee-zl" viewBox="0 0 48 48">
+        <defs/>
+        <g clip-path="url(#clip0)">
```

Comment:
Not sure if these svgs were ran through an optimizer (such as SVGOMG), but looks like we could and should do some additional cleanup.